src/main/java/org/openrewrite/java/migrate/guava/NoGuavaCreateTempDir.java

```diff
@@ -0,0 +1,140 @@
+package org.openrewrite.java.migrate.guava;
+
+import org.openrewrite.Cursor;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.search.UsesMethod;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.tree.*;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class NoGuavaCreateTempDir extends Recipe {
+    @Override
+    public String getDisplayName() {
+        return "Use java.io.Files.createTempDirectory instead of Guava";
```

Comment:
Please use ticks to surround the method pattern. No need for it to be fully qualified. So `Files#createTempDirectory()`